### ISSUE TITLE
Easy install/bootstrap script

### ISSUE THREAD

**Issue by wolph** [NewRelic-Python-Plugins/newrelic-python-agent#14]
Looking at the readme I think perhaps this is something that could (should?) be bootstrapped by a script of some sort. Manually creating users, config files, directories, etc... seems a tedious process.

Should be easy enough to do from Python, ideally I'd like to avoid shell scripting as there is no real need and Python is available on pretty much all systems anyhow.

Inspiration: https://bootstrap.pypa.io/get-pip.py

**Comment by wolph** [comment]
It should be noted that even `pip` is not guaranteed to be available on all systems so I'm not sure if we can rely on that...

**Comment by Rockstar04** [comment]
When I just went through a fresh install of the plugin on my U 16.04 (minimal) I had to install PIP first. I really like the idea of automating as much as possible, but we need to look into the possible differences in file location in REHL vs Debian. 

Also as noted in the readme I just updated, the newrelic user would already be created if you installed an APM or NewRelic Server monitoring plugin already.
